app/controllers/twilio_webhooks_controller.rb

```diff
@@ -15,13 +15,20 @@ def update_outbound_call
     update_params[:call_duration] = params["CallDuration"] if params["CallDuration"].present?
     call.update(update_params)
   end
-
+  
   def dial
```

Comment:
possibly silly question: why is it that `OutboundCallForm` has a `dial` method and so does `TwilioWebhooksController` and so does `Twilio::TwiML::VoiceResponse`? seems a bit confusing

Reply:
OutboundCallForm has a dial method that dials the call from our twilio number to the VITA User. When we instantiate the call object, we pass it the `dial` method from the TwilioWebhooksController (which I renamed to dial_client and added a comment to hopefully help with the confusion), which is a method that Twilio will post to when the call with the User connects so that we can then connect them to the client. The `Twilio::TwiML::VoiceResponse` dial builds a dial node into the XML response so that twilio knows to call the user.